vllm/config.py

```diff
@@ -392,7 +393,8 @@ def _verify_quantization(self) -> None:
             quant_method = quant_cfg.get("quant_method", "").lower()
 
             # Detect which checkpoint is it
-            for _, method in QUANTIZATION_METHODS.items():
+            for name in QUANTIZATION_METHODS:
+                method = get_quantization_config(name)
```

Comment:
@DarkLight1337 this code already tests all registered quantization methods have the corresponding config, and the lazy import works.

Reply:
Oh, I missed this, thanks for pointing it out!